src/random.jl

```diff
@@ -114,3 +114,69 @@ Return BitVector of primary producers.
 function whoisproducer(A)
     vec(.!any(A, dims=2))
 end
+
+"""
+**Create facilitation links**
+
+Takes trophic matrix (A) and number or connectance of facilitation links (resp. L and C).
+Returns matrix of facilitation links.
+Facilitating species are in rows, facilitated species are in columns.
+"""
+function facilitationlinks(A, L::Int64)
```

Comment:
I'm not sure whether there is an advantage to require that `L` be of exact type `Int64`. Is there any reason you did not choose `facilitationlinks(A, L)` instead?